### ISSUE TITLE
Don't find input fields, only on the bower installation.

### ISSUE THREAD

**Issue by RonenNess** [callumacrae/JSColor#1]
I noticed that the jscolor I get when using bower install ("bower install JSColor --save") does not work on the following element:

`<p>Color: <input id="background-color" class="jscolor {onFineChange:'updateBackground(this)'}" value="6A9AAB"></p>`

Which should be valid + works with the minified version I got from the site (the zip you can download from http://jscolor.com/)

I debugged it and saw that in this line in bind():

`if(!e[i].color && e[i].className && (m = e[i].className.match(matchClass))) {`

It fails on the condition:

`e[i].className.match(matchClass)`

So I compared this part with the minified version I had from the site (I edited the names etc in the minified so it will be easier to compare):

```
var matchClass =    new RegExp('(^|\\s)('+jscolor.bindClass+')\\s*(\\{[^}]*\\})?', 'i');   // from bower
var r =             new RegExp('(^|\\s)('+jscolor.bindClass+')(\\s*(\\{[^}]*\\})|\\s|$)','i');  // minified
```

as you can see the regex is slightly different which I guess is the problem.
## tl;dr

The bower version of jscolor have a wrong regex that don't identify valid input fields that are shown as examples in the site and are successfully identified by the (probably older) minified zip version.

Awesome lib btw :+1: 

Thanks!

**Comment by callumacrae** [comment]
Hey!

I actually have no memory of this project: it's from a few years ago. I have no plans to support it further as I can't remember how it works!

This repo is just a mirror of the original repo, with an added bower.json. The original is here (maybe it would be a better place for it): https://github.com/EastDesire/jscolor

Sorry for not being able to help! This was a super detailed description of the issue.

**Comment by RonenNess** [comment]
No worries, but please consider updating the bower package, because right now its broken (which means I have to include jscolor js in my project's git and its a bad practice).

Thanks :)
